client/src/app/site/motions/modules/motion-block/components/motion-block-list/motion-block-list.component.ts

```diff
@@ -127,21 +134,29 @@ export class MotionBlockListComponent extends BaseListViewComponent<ViewMotionBl
     }
 
     /**
-     * Helper function reset the form and set the default values
+     * Helper function reset the form and set the default values as well as closing the modal dialog
      */
     public resetForm(): void {
         this.createBlockForm.reset();
         this.createBlockForm.get('agenda_type').setValue(this.defaultVisibility);
+        if (this.dialogRef) {
```

Comment:
Closing the dialog is caused by the dialog itself. This is not required here.
Furthermore, there is no GUI-element (anymore) that accesses `resetForm()`. Please set to private